### ISSUE TITLE
Accelerator backward with retain graph

### ISSUE THREAD

**Issue by minhlong94** [huggingface/accelerate#103]
I want to use `loss.backward(retain_graph=True)` instead of normal backward. How can I do this using accelerate?

**Comment by sgugger** [comment]
Ah, this is currently not implemented, will make a PR to add support for this, good point!
